blog/2019-12-20-infinispan-10.1.0.Final.adoc

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog
+title: Infinispan 10.1.0.Final
+date: '2019-12-20T12:00:00.000-00:00'
+author: Tristan Tarrant
+tags: [ "release" ]
+---
+
+= Infinispan 10.1 "Turia"
+
+Hi there,
+
+we finish 2019 in style with the Final release of Infinispan 10.1, codenamed https://www.cervezaturia.es["Turia"].
+
+== Server console
+
+The highlight of this release is the new server console which is now based on https://www.patternfly.org/v4/[Patterfly 4] and https://www.patternfly.org/v4/[React.js].
+We will soon have a blog post detailing the work that has been done and our future plans. In the meantime, here are a few screenshots:
+
+[caption="Welcome",link=/blog/img/ispn101welcome.png]
+image::/blog/thumb/ispn101welcome.png[Welcome Page]
+
+[caption="Console: caches",link=/blog/img/ispn101console.png]
+image::/blog/thumb/ispn101console.png[Console Caches]
+
+[caption="Console: cache stats",link=/blog/img/ispn101console2.png]
+image::/blog/thumb/ispn101console2.png[Console Cache Stats]
+
+== Security
+
+Many changes related to security have happened since 10.0:
+
+* Native SSL/TLS provided by https://github.com/wildfly-security/wildfly-openssl[WildFly OpenSSL]. The server only ships with native libraries for Linux x86_64, but you can download natives for other platforms and architectures
+* Improved usability of client configuration with better defaults
+* Full support for authorization for admin operations via the RESTful endpoint
+* Console authentication support
+* Kerberos authentication for both Hot Rod (GSSAPI, GS2) and HTTP/Rest (SPNEGO)
+* Improved LDAP realm configuration with connection tuning and attribute references
+* Rewritten client/server security documentation including examples on how to create certificate chains, connecting to various LDAP directories and https://www.keycloak.org/[KeyCloak], etc.
+
+== Server
+
+* A command-line switch to specify an alternate logging configuration file
+* Query and indexing operations/stats are now exposed over the RESTful API
+* Tasks and Scripting support
+* Support for binding the endpoints to 0.0.0.0 / ::0 (aka INADDR_ANY)
+
+== Non-blocking
+
+More work has landed on the quest to completely remove blocking calls from our internals. The following have been made non-blocking:
+
+* state transfer
```

Comment:
start the sentence with a capital letter? 
same for below

Reply:
Done